src/langchain_google_memorystore_redis/__init__.py

```diff
@@ -12,6 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 from langchain_google_memorystore_redis.chat_message_history import MemorystoreChatMessageHistory
+from langchain_google_memorystore_redis.doc_saver import MemorystoreDocumentSaver
```

Comment:
Can we use the relative path instead? it is more resilient to potential name changes in the future